lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -248,23 +248,23 @@ <h3 class="hd hd-3">${_("Course Team Management")}</h3>
       data-add-button-label="${_("Add Moderator")}"
     ></div>
 
-    <div class="auth-list-container"
+    <div class="auth-list-container group_moderator"
       data-rolename="Group Moderator"
-      data-display-name="${_("Discussion Group Moderators")}"
+      data-display-name="${_("Discussion Group Community TA")}"
       data-info-text="
-        ${_("Discussion Group Moderators can edit or delete any post, clear misuse flags, close "
-         "and re-open threads, endorse responses, and see posts from all groups. "
-         "Their posts are marked as 'staff'. They cannot manage course team membership by "
-         "adding or removing discussion moderation roles. Only enrolled users can be "
-         "added as Discussion Moderators.")}"
+        ${_("Discussion Group Community TAs are members of the community whom you deem particularly "
+        "helpful on the discussion boards. They can edit or delete any post, clear "
+        "misuse flags, close and re-open threads, endorse responses, and see posts from "
+        "only their assigned groups. Their posts are marked as 'Community TA'. Only enrolled users can "
+        "be added as Community TAs.")}"
       data-list-endpoint="${ section_data['list_forum_members_url'] }"
       data-modify-endpoint="${ section_data['update_forum_role_membership_url'] }"
-      data-add-button-label="${_("Add Group Moderator")}"
+      data-add-button-label="${_("Add Group Community TA")}"
     ></div>
 
     <div class="auth-list-container"
       data-rolename="Community TA"
-      data-display-name="${_("Discussion Community TAs")}"
+      data-display-name="${_("Discussion Community TA")}"
```

Comment:
Can we please update the old text for Community TAs as well:
"Community TAs are members of the community who help course teams moderate discussions. They can see posts by all learners, and can edit or delete posts, clear flags, close and re-open threads, and endorse responses. Their posts are marked..."